WooCommerce/Classes/ViewRelated/Dashboard/Settings/In-Person Payments/Payments Menu/SimplePaymentsMigrationView.swift

```diff
@@ -0,0 +1,74 @@
+import SwiftUI
+
+/// Shows information about the migration from simple payments to order creation in Menu tab > Payments > Collect Payment.
+struct SimplePaymentsMigrationView: View {
+    let addCustomAmount: () -> Void
+
+    var body: some View {
+        VStack(spacing: Layout.defaultVerticalSpacing) {
+            VStack(spacing: Layout.verticalSpacingBetweenElements) {
+                Image(uiImage: .bell)
+                Text(Localization.title)
+                    .titleStyle()
+                    .bold()
+                    .fixedSize(horizontal: false, vertical: true)
+            }
+            VStack(spacing: Layout.verticalSpacingBetweenElements) {
+                Text(.init(Localization.subtitle))
+                    .bodyStyle()
+                Text(Localization.detail)
+                    .foregroundColor(Color(.secondaryLabel))
+                    .bodyStyle()
+            }
+            Color.clear
+                .frame(height: 8)
+            Button(Localization.addCustomAmount) {
+                addCustomAmount()
+            }
+            .buttonStyle(PrimaryButtonStyle())
```

Comment:
Would maybe setting top padding on button be a better way? Instead of 8pt Color.clear?

```
Button(Localization.addCustomAmount) {
                addCustomAmount()
            }
.padding(.top, 8.0)
```

Reply:
Looks nicer than a spacer, updated in https://github.com/woocommerce/woocommerce-ios/pull/12400/commits/9d1d3fd5d1dda0109845a3793e971456743ec64f and tested that the spacing looks the same